### PR TITLE
add generic project minter config events

### PR DESCRIPTION
**Pull request by ryley-o**
This updates our FilteredMinter interface to include 20 events representing generic project minter config instructions.

Note: currently getting ethers nuisance warnings during tests due to overloaded events, working to find a solution to silence warnings...

### CODE REVIEW

File: contracts/interfaces/0.8.x/IFilteredMinterV1.sol [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+     * for project `_projectId`.
+     */
+    event ConfigKeyRemoved(uint256 indexed _projectId, bytes32 _key);
+
+    /// BOOL
+    /**
+     * @notice Generic project minter configuration event. Sets value of key
+     * `_key` to `_value` for project `_projectId`.
+     */
+    event ConfigValueSet(uint256 indexed _projectId, bytes32 _key, bool _value);
```
[the remark this answers is not in the document]
Comment (ryley-o) [reply contracts/interfaces/0.8.x/IFilteredMinterV1.sol:28]: I think I will keep `_value` here in the case that some minter might want to utilize a nullable bool field. These events are minimally triggered, so additional gas cost is not very troublesome.

**Review by jakerockland — APPROVED** [review]
Ah shoot, my apologies, I had thought I already approved this one! 🤦‍♂️

**Comment by ryley-o** [comment]
gentle ping to @jakerockland on this one, I feel good about merging if you agree.

**Comment by ryley-o** [comment]
@jakerockland okay pretty funny how this sat this long 😆 then we comment within 30 seconds of each other! merging...

**Comment by ryley-o** [comment]
no ser you were not a blocker - been bogged down with other stuff and just noticed this one sitting 👍

**Pull request merged**